### PR TITLE
Switch from Nextjs to Create-React-App (no SSR)

### PR DESCRIPTION
**Pull request by mondoreale**
A couple of things happen in this PR:
- There's no SSR anymore, we roll with client-generated single-page app using `HashRouter` from `react-router-dom`
  - `/` - previous index/home page
  - `/#/chat` - previous chat (`/chat`)
- Introduced `styled-components` and rewrote most of the previous stuff
- Fixed fonts (using the "web" variants now)
- Implemented very basic `Message` component (2 variants: incoming / outgoing)
- Implemented basic message handling (write a message and hit `Enter`). Feed element (list of messages) auto-scrolls to the most recent message every time message collection updates.

Screen Shot 2021-12-17 at 10 03 59 PM

It's still in the works but I thought you may wanna have a look already. @mxinburritos @bearni95 

A very simple `npm ci && npm start` will get you going.

### CODE REVIEW

**Comment by peetu** [comment]
Any netlify related check can be ignored, I will disconnect netlify as soon as the site builds somehere else we can access.

File: package.json [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
+    "@types/react-helmet": "^6.1.4",
+    "@types/react-router-dom": "^5.3.2",
+    "@types/styled-components": "^5.1.18",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-helmet": "^6.1.0",
+    "react-router": "^6.1.1",
+    "react-router-dom": "^6.1.1",
+    "react-scripts": "5.0.0",
+    "streamr-client": "^6.0.0-alpha.17",
```
Comment (fonty1) [inline package.json:26]:
```suggestion
    "streamr-client": "^6.0.0-alpha.19",
```

File: package.json [hunk 2]
```diff
@@ -1,40 +1,55 @@
 {
-  "name": "streamr-chat-next",
+  "name": "streamr-chat2",
```
Comment (fonty1) [inline package.json:2]:
```suggestion
  "name": "the-chat-app",
```

File: public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (fonty1) [inline public/manifest.json:2]:
```suggestion
  "short_name": "The Chat App",
```

File: public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "Create React App Sample",
```
Comment (fonty1) [inline public/manifest.json:3]:
```suggestion
  "name": "The Chat App,
```

File: src/components/pages/Chat/Message.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+`
+
+const Avatar = styled.div`
+    background-color: #F1F4F7;
+    border-radius: 50%;
+    height: 2.5rem;
+    width: 2.5rem;
+`
+
+const ME = '0xa047314ff08Cead7726Ceb90fc0CB39D80607227'
```
Comment (fonty1) [inline src/components/pages/Chat/Message.tsx:31]: Not sure what this is, @bearni95 any ideas?

**Review by fonty1 — APPROVED** [review]
Merge as you please :) This is a yuge step up!

File: src/components/pages/Chat/Message.tsx (hunk 5 above)
Comment (mondoreale) [reply src/components/pages/Chat/Message.tsx:31]: It's "work in progress". ;)

**Comment by mondoreale** [comment]
@fonty1 I've included you change suggestions! gh actions + pages are pre-set up (will get triggered after a merge). I'll add another round of tweaks and fixes to it and merge in tonight!

Cheers guys!

**Pull request merged**